www/components/Loading.tsx

```diff
@@ -1,16 +1,14 @@
 // ported from https://github.com/suren-atoyan/monaco-react
 
 const loadingStyles = {
-  display: 'flex',
-  height: '100%',
-  width: '100%',
-  justifyContent: 'center',
-  alignItems: 'center',
+  display: "flex",
+  height: "100%",
+  width: "100%",
+  justifyContent: "center",
+  alignItems: "center",
 };
```

Comment:
```suggestion
flex h-full w-full justify-center items-center
```

Why are we not using twind here?